[2024-08-21T20:45:45.690+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-21T20:45:45.711+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 3_models_dag_home.init manual__2024-08-21T15:45:43.574001+00:00 [queued]>
[2024-08-21T20:45:45.719+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 3_models_dag_home.init manual__2024-08-21T15:45:43.574001+00:00 [queued]>
[2024-08-21T20:45:45.719+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-21T20:45:45.745+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): init> on 2024-08-21 15:45:43.574001+00:00
[2024-08-21T20:45:45.759+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=303387) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T20:45:45.760+0500] {standard_task_runner.py:72} INFO - Started process 303457 to run task
[2024-08-21T20:45:45.759+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '3_models_dag_home', 'init', 'manual__2024-08-21T15:45:43.574001+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/homework_stepik-checkpoint.py', '--cfg-path', '/tmp/tmp3m472nn_']
[2024-08-21T20:45:45.761+0500] {standard_task_runner.py:105} INFO - Job 175: Subtask init
[2024-08-21T20:45:45.824+0500] {task_command.py:467} INFO - Running <TaskInstance: 3_models_dag_home.init manual__2024-08-21T15:45:43.574001+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-21T20:45:45.906+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='3_models_dag_home' AIRFLOW_CTX_TASK_ID='init' AIRFLOW_CTX_EXECUTION_DATE='2024-08-21T15:45:43.574001+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-21T15:45:43.574001+00:00'
[2024-08-21T20:45:45.907+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-21T20:45:46.167+0500] {migration.py:216} INFO - Context impl PostgresqlImpl.
[2024-08-21T20:45:46.168+0500] {migration.py:219} INFO - Will assume transactional DDL.
[2024-08-21T20:45:46.211+0500] {python.py:240} INFO - Done. Returned value was: {'start_tiemstamp': '20240821 20:45'}
[2024-08-21T20:45:46.240+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-21T20:45:46.240+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=3_models_dag_home, task_id=init, run_id=manual__2024-08-21T15:45:43.574001+00:00, execution_date=20240821T154543, start_date=20240821T154545, end_date=20240821T154546
[2024-08-21T20:45:46.295+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-21T20:45:46.323+0500] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-21T20:45:46.325+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
